src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,12 +34,18 @@ internal TagHelperOutput(string tagName)
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
         public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+            : this(tagName, attributes, HtmlEncoder.Default)
```

Comment:
@NTaylorMullen the `IHtmlHelper` comes from DI, not this `static` property in any real scenario.

my point is users and authors are most often not the same.  us as framework authors creating the MVC tag helpers and our customers is just one example of the separation.  basically what you're making "easy" is creation of a bunch of tag helpers that use an `IHtmlEncoder` implementation that application authors can't override.